From 272f19cff1484a22f16f33d4e67d16120dff19bd Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Wed, 12 Jul 2023 09:59:01 +0200
Subject: [PATCH 0431/1141] ARM: dts: stm32: revert reboot mode support to
 stm32mp157

This reverts commit c9459b65bf21325f6b7627a04bda5845ab7ffd5f.

We remove the reboot mode based on syscon as we will introduce NVMEM API
for access to the TAMP backup registers. The reboot mode will use
nvmem-reboot-mode instead of syscon-reboot-mode

Change-Id: Ic3da3f12a2219f5a3d1ccaa8c18415963873b517
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
---
 arch/arm/boot/dts/stm32mp151.dtsi | 13 -------------
 1 file changed, 13 deletions(-)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1952,19 +1952,6 @@
 		tamp: tamp@5c00a000 {
 			compatible = "st,stm32-tamp", "syscon", "simple-mfd";
 			reg = <0x5c00a000 0x400>;
-
-			reboot-mode {
-				compatible = "syscon-reboot-mode";
-				offset = <0x150>; /* reg20 */
-				mask = <0xff>;
-				mode-normal = <0>;
-				mode-fastboot = <0x1>;
-				mode-recovery = <0x2>;
-				mode-stm32cubeprogrammer = <0x3>;
-				mode-ums_mmc0 = <0x10>;
-				mode-ums_mmc1 = <0x11>;
-				mode-ums_mmc2 = <0x12>;
-			};
 		};
 
 		/*
